examples/30_extended/create_upload_tutorial.py

```diff
@@ -100,8 +100,8 @@
     # The attribute that represents the row-id column, if present in the
     # dataset.
     row_id_attribute=None,
-    # Attributes that should be excluded in modelling, such as identifiers and
-    # indexes.
+    # Array of attributes that should be excluded in modelling, such as 
+    # identifiers and indexes. E.g. ["feat1","feat2"]
```

Comment:
```suggestion
    # Attribute or list of attributes that should be excluded in modelling, such as 
    # identifiers and indexes. E.g. "feat1" or ["feat1","feat2"]
```

Reply:
My suggestion to reflect that it can be either a single attribute or a list of attributes (and change "array" to "list").